extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -173,7 +150,7 @@ Implicit tiling only requires defining the subdivision scheme, refinement strate
 | --- | --- |
```

Comment:
should we add a row in this table to say that `extensions` is constant for all descendant tiles? or should we not define this except in specific cases like for multiple contents?